### DISCUSSION TITLE
Property does not exist on type 'LucidRow | LucidRow[]' on prelaoded relations

### DISCUSSION THREAD

**Discussion by hsharma1996** [adonisjs/core#3203] [Help]
When I try to modify/access value on a prelaoded model, ts highlight this issue, Can you please guide what I am doing wrong ?

**Comment by hsharma1996** [comment]
@thetutlage Sorry for tag, I tried researching but unable to sort this out. I am trying to access/update property on preloaded relations.

**Comment by thetutlage** [comment]
Can't help without looking at any code.

**Reply by hsharma1996** [reply]
@thetutlage I am writing short code, Hopefully that will to help. 
`let team_1 = match.$preloaded.team_1;`
`let selectedTeam = team_1.shortName`

This is how I am trying to access shortName on my team model which I prelaoded with match model.

`@column()
  public shortName: string | null`

This is how I am declaring the column inside the model. 

hovering over `team_1.shortName` says `Property 'shortName' does not exist on type 'LucidRow | LucidRow[]'.
  Property 'shortName' does not exist on type 'LucidRow`

**Reply by WeyIin** [reply]
Where and how do you preload?
Try `match.team_1` ?

**Reply by hsharma1996** [reply]
> Where and how do you preload? Try `match.team_1` ?

I was actually accessing $preloaded which are lucid data, match.team_1 is what solved the issue. Thanks man.

**Reply by WeyIin** [reply]
excellent, mark it as answered :)
